inc/class_datacache.php

```diff
@@ -687,15 +684,14 @@ function update_statistics()
 		');
 		$topposter = $db->fetch_array($query);
 
-		$query = $db->simple_select('users', 'COUNT(*) AS posters', 'postnum>0');
+		$query = $db->simple_select('users', 'COUNT(uid) AS posters', 'postnum>0');
 		$posters = $db->fetch_field($query, 'posters');
 
 		$statistics = array(
 			'time' => TIME_NOW,
-			'top_forum' => (array)$forum,
 			'top_referrer' => (array)$topreferrer,
 			'top_poster' => (array)$topposter,
-			'posters' => (int)$posters,
```

Comment:
I'm not trying to discuss anything really. I'm concerned more about this fixing multiple bugs in the same PRs, having 11 files changed and most of the edits are because of why not. It makes it harder to test the code. Just saying.

I actually agree some changes are more readable now even when I wrote most of it. But, well I already said it.